.buildkite/bk.integration-fips.pipeline.yml

```diff
@@ -0,0 +1,142 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
+  VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
```

Comment:
I’d like to clarify the usage of this secret, as it appears to be exclusively associated with the cleanup step and pipeline:

- https://github.com/search?q=repo%3Aelastic%2Felastic-agent%20VAULT_PATH&type=code
- https://github.com/search?q=repo%3Aelastic%2Felastic-agent+gce-cleanup.sh&type=code

I'm not familiar with the background here, @pazone, could you please double-check this? Thanks!

Reply:
That's right. `VAULT_PATH` is not needed here and in a few other pipelines. I see other minor issues in this and other pipelines. I assigned a cleanup issue to myself

Reply:
Well... if it's not needed, I'd suggest to clean this up here instead adding it

```suggestion
```